### DISCUSSION TITLE
Article tag?

### DISCUSSION THREAD

**Discussion by simplerethink** [withastro/starlight#2328] [Q&A] [answered]
Is there a way to change starlight templates to use an `article` tag? Currently, the article content is in `main`, but I'd rather have `main` used for the content + TOC and wrap the content with `article`. Wouldn't a structure like this be more semantic?

**Comment by Fryuni** [comment]
You can override the [`MarkdownContent` component](https://starlight.astro.build/reference/overrides/#markdowncontent) to use an `<article>` instead of a `<div>`

**Comment by delucis** [comment, marked as answer]
The `<article>` element doesn’t really have much in the way of additional semantics. It was introduced with HTML 5 with a lot of fanfare about semantics, but it doesn’t do a whole lot. Most often it is used on pages with _multiple_ pieces of content, e.g. a blog index, a grid of product cards, etc. But in Starlight’s case, I don’t believe it would make any meaningful difference?
